content/en/docs/tasks/administer-cluster/migrating-from-dockershim/ migrating-telemetry-and-security-agents.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Migrating telemetry and security agents from dockershim
+content_type: task 
+reviewers:
+- SergeyKanzhelev
+- jrcamp # for SignalFx (Splunk)
+weight: 70
+---
+
+<!-- overview -->
+
+Docker is being deprecated. From the [Dockershim Deprecation
+FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/) you might already
+know that most apps do not have a direct dependency on runtime hosting
+containers. However, there are still a lot of telemetry and security agents
+that has a dependency on docker to collect containers metadata, logs and
+metrics. This document aggregates information on how to detect these
+dependencies and links on how to migrate these agents to use generic tools or
+alternative runtimes.
```

Comment:
To help pages stand the test of time, try to write wording that works both during and after the deprecation. It's not always possible but when it can be done it avoids work to track and reword things after the fact.

I've seen quite a few dated pages on the Kubernetes docs so I try to word assuming that the cleanup probably gets forgotten.